<ui version="4.0" >
 <class>dEditCfg</class>
 <widget class="Q3MainWindow" name="dEditCfg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>364</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>400</width>
    <height>364</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Business scheme parameters</string>
  </property>
  <widget class="QWidget" name="widget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>461</width>
     <height>364</height>
    </rect>
   </property>
   <layout class="QGridLayout" >
    <item row="0" column="0" >
     <widget class="QTabWidget" name="tabWidget" >
      <property name="font" >
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <widget class="QWidget" name="tab" >
       <attribute name="title" >
        <string>Common</string>
       </attribute>
       <layout class="QVBoxLayout" >
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QLabel" name="textLabel5" >
            <property name="minimumSize" >
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="frameShape" >
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Plain</enum>
            </property>
            <property name="text" >
             <string>Name</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="eName" />
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QLabel" name="textLabel7" >
            <property name="minimumSize" >
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="text" >
             <string>Author</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="eAuthor" />
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QLabel" name="textLabel8" >
            <property name="minimumSize" >
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="text" >
             <string>Date</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="eDate" />
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QLabel" name="textLabel6" >
            <property name="text" >
             <string>Description</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Q3TextEdit" name="eDescription" />
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab" >
       <attribute name="title" >
        <string>User</string>
       </attribute>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="QLabel" name="textLabel1" >
          <property name="minimumSize" >
           <size>
            <width>160</width>
            <height>0</height>
           </size>
          </property>
          <property name="text" >
           <string>Data base server user</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QLineEdit" name="eDBUserName" />
        </item>
        <item row="1" column="0" colspan="2" >
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QLabel" name="textLabel2" >
            <property name="minimumSize" >
             <size>
              <width>160</width>
              <height>0</height>
             </size>
            </property>
            <property name="text" >
             <string>Data base password</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="eDBPass" >
            <property name="echoMode" >
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0" colspan="2" >
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QLabel" name="textLabel3" >
            <property name="minimumSize" >
             <size>
              <width>160</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>160</width>
              <height>32767</height>
             </size>
            </property>
            <property name="text" >
             <string>Data base type</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="eDBType" >
            <property name="editable" >
             <bool>false</bool>
            </property>
            <property name="duplicatesEnabled" >
             <bool>false</bool>
            </property>
            <item>
             <property name="text" >
              <string>Unknown</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>mysql</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>postgres</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="0" colspan="2" >
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QLabel" name="textLabel4" >
            <property name="minimumSize" >
             <size>
              <width>160</width>
              <height>0</height>
             </size>
            </property>
            <property name="text" >
             <string>Data base server address</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="eDBHost" />
          </item>
         </layout>
        </item>
        <item row="5" column="0" colspan="2" >
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QLabel" name="textLabel6_2" >
            <property name="minimumSize" >
             <size>
              <width>160</width>
              <height>0</height>
             </size>
            </property>
            <property name="text" >
             <string>Data base name</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="eDBName" />
          </item>
         </layout>
        </item>
        <item row="4" column="0" colspan="2" >
         <layout class="QHBoxLayout" >
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLabel" name="textLabel5_2" >
              <property name="minimumSize" >
               <size>
                <width>160</width>
                <height>0</height>
               </size>
              </property>
              <property name="text" >
               <string>Data base server port</string>
              </property>
              <property name="wordWrap" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="eDBPort" />
            </item>
           </layout>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType" >
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>130</width>
              <height>21</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="7" column="0" colspan="2" >
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QLabel" name="textLabel7_2" >
            <property name="minimumSize" >
             <size>
              <width>160</width>
              <height>0</height>
             </size>
            </property>
            <property name="text" >
             <string>Work directory</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="eWorkDir" />
          </item>
         </layout>
        </item>
        <item row="8" column="0" colspan="2" >
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QLabel" name="textLabel8_2" >
            <property name="minimumSize" >
             <size>
              <width>160</width>
              <height>0</height>
             </size>
            </property>
            <property name="text" >
             <string>Configuration file name</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="eCfgName" />
          </item>
         </layout>
        </item>
        <item row="9" column="1" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="6" column="0" >
         <widget class="QLabel" name="textLabel1_2" >
          <property name="text" >
           <string>Data base header</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="6" column="1" >
         <widget class="QLineEdit" name="eDBHeader" />
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab" >
       <attribute name="title" >
        <string>Global module</string>
       </attribute>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="QSEditor" native="1" name="eModule" />
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="5" margin="5" />
 <pixmapfunction>rcIcon</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3MainWindow</class>
   <extends>QWidget</extends>
   <header>q3mainwindow.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3TextEdit</class>
   <extends>Q3Frame</extends>
   <header>q3textedit.h</header>
  </customwidget>
  <customwidget>
   <class>QSEditor</class>
   <extends>QWidget</extends>
   <header>qseditor.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local" >mdtree.h</include>
  <include location="local" >acfg.h</include>
  <include location="local" >qseditor.h</include>
  <include location="local" >mainform.h</include>
  <include location="local" >qseditor.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
